### DISCUSSION TITLE
Group flyway dependencies by default

### DISCUSSION THREAD

**Discussion by keks42** [renovatebot/renovate#30247] [Suggest an Idea]
### Tell us more.

In version 10 flyway modularized the database dependencies. [1, 2]

> Modularized database support in Flyway to allow greater flexibility. This includes; DB2, Derby, HSQLDB, Informix, PostgreSQL, CockroachDB, Redshift, SAP HANA, Snowflake and Sybase ASE. See Database Support page for your database for module dependency. If you are including Flyway in your project, either as a dependency or via the maven and gradle plugins please include the respective database module in your project configuration.

As it is common to use flyway in combination with spring and spring is grouped by default, it makes sense to group flyway dependencies by default too.

Proposal: Group flyway dependencies by default.

[1] https://documentation.red-gate.com/flyway/release-notes-and-older-versions/release-notes-for-flyway-engine
[2] https://documentation.red-gate.com/fd/flyway-v10-has-landed-222627771.html

**Comment by rarkins** [comment]
If they're from the same repo then they can be added to group:monorepo, otherwise group:recommended

**Reply by keks42** [reply]
Thanks for the fast reply! 
I have opened https://github.com/renovatebot/renovate/pull/30248
